packages/cdktf-cli/bin/cmds/ui/models/terraform-cloud.ts

```diff
@@ -127,6 +130,8 @@ export class TerraformCloud implements Terraform {
 
     this.configurationVersionId = version.id
 
+    this.removeLocalTerraformDirectory()
```

Comment:
There are two modes for Terraform Cloud: local execution and remote execution. Could this possibly break local execution or be unwanted (as it destroys the local cache containing already initialized Terraform providers)? 🤔

Reply:
I would guess no as this is the terraform cloud implementation of this function. The Cli implementation would need to be able to deal with a blank state so it should work IMHO

Reply:
Yeah, I think this should be ok - since that's scoped to the stack anyway. Even if you had multiple stacks targeting local and cloud, it should still be ok.